importer/SeBbrSv.py

```diff
@@ -81,15 +82,6 @@ def set_heritage_bbr(self):
             qualifier = None
         self.add_statement("heritage_status", type_q, qualifier)
```

Comment:
the reference for this should be kulturarvsdata ideally something like:
* dataset= BBR/kulturarvsdata
* id = kulturarvsdata_id
* url = url
* retrieval date = today